src/imitation/policies/interactive.py

```diff
@@ -0,0 +1,84 @@
+import abc
+from typing import Optional, List
+
+import gym
+import matplotlib.pyplot as plt
+import numpy as np
+
+import imitation.policies.base as base_policies
+from imitation.util import util
+
+
+class DiscreteInteractivePolicy(base_policies.NonTrainablePolicy, abc.ABC):
+    def __init__(
+        self,
+        observation_space: gym.Space,
+        action_space: gym.Space,
+        action_names: List[str],
+        action_keys: List[str],
+        clear_screen_on_query: bool = True,
+    ):
+        super().__init__(
+            observation_space=observation_space,
+            action_space=action_space,
+        )
+
+        assert isinstance(action_space, gym.spaces.Discrete)
+        assert len(action_names) == len(action_keys) == action_space.n
+        # Names and keys should be unique.
+        assert len(set(action_names)) == len(set(action_keys)) == action_space.n
```

Comment:
Yeah, I did this because of ordering, and had doubts about enforcing `OrderedDict` type, but now I think I agree it's more elegant so changing into `OrderedDict`.